NBitcoin/Bitcoin.cs

```diff
@@ -88,10 +88,13 @@ private static uint GetSignetMagic()
 
 		public Network Testnet => Network.TestNet;
 
+		public Network Testnet4 => Network.TestNet4;
+
 		public Network Regtest => Network.RegTest;
 
 		public string CryptoCode => "BTC";
 
+		public static readonly ChainName TestNet4Name = new ChainName("TestNet4");
```

Comment:
This change is why I took so long to push: I'm not sure if it's correct to have this `public`. I tried another approach and move `TestNet4` to `Bitcoin` class but ended up with a similar (worse probably) problem, [see](https://github.com/MetacoSA/NBitcoin/commit/abc2aec594d02a15e2e254fec8858002e0c95ab7#diff-f8a547ef70a0b0926b695ed245d377c50c5052ca773f4c418a8b334d0cee929dR1911)